docs/dagster-university/pages/dagster-essentials/extra-credit/materialization-metadata.md

```diff
@@ -14,8 +14,8 @@ Now that we’ve covered definition metadata, let’s dive into the other type o
 
 To add metadata to an asset, you need to do two things:
 
-- Use the `context.add_output_metadata` function to pass in the data
 - Use the `MetadataValue` utility class to wrap the data, ensuring it displays correctly in the UI
+- Return a `MaterializeResult` instance with the `metadata` parameter from your asset
```

Comment:
Nit - this comes first in code, so lets have the order of this list match that.

Reply:
Also, is it `instance` or `object`?

Reply:
From what I'm seeing online, `instance` and `object` are mostly interchangeable. I've changed it to `object` to be aligned with the rest of the documentation.